iphone/Classes/TiUIWebViewProxy.m

```diff
@@ -262,40 +267,40 @@ - (NSDictionary *)backForwardList
 - (NSDictionary *)preferences
 {
   return @{
-    @"minimumFontSize" : NUMFLOAT([[[[[self webView] webView] configuration] preferences] minimumFontSize]),
-    @"javaScriptEnabled" : NUMBOOL([[[[[self webView] webView] configuration] preferences] javaScriptEnabled]),
-    @"javaScriptCanOpenWindowsAutomatically" : NUMBOOL([[[[[self webView] webView] configuration] preferences] javaScriptCanOpenWindowsAutomatically]),
+    @"minimumFontSize" : NUMFLOAT([[[[self wkWebView] configuration] preferences] minimumFontSize]),
```

Comment:
All these brackets in all these changes scream out to me to open this in Xcode and use dot notation if possible, i.e.:
```suggestion
    @"minimumFontSize" : NUMFLOAT(self.wkWebView.configuration.preferences.minimumFontSize),
```

Just looks less messy to my eye and easier to follow. Not required (and possibly, not possible!)